src/commands/skuid/page/pull.ts

```diff
@@ -114,16 +117,17 @@ export default class Pull extends SfdxCommand {
       // Clear out memory from our response
       delete skuidPages[pageName];
     });
-    this.ux.log('Wrote ' + numPages + ' pages to ' + dir);
 
-    return {};
+    this.log('Wrote ' + numPages + ' pages to ' + dir);
+
+    return 'Wrote ' + numPages + ' pages to ' + dir;
```

Comment:
@jt-deane I don't think I should be doing this but it helped to pass tests. I do think we should revert it back to `return {};` and in the tests we need to figure out how to access what was logged when we call `this.log()`.